src/Fractions/Fraction.DebugProxy.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Diagnostics;
+using System.Globalization;
+using System.Numerics;
+using Fractions.Formatter;
+
+namespace Fractions;
+
+public readonly partial struct Fraction {
+    internal readonly struct FractionDebugView(Fraction fraction) {
+        public BigInteger A => fraction.Numerator;
+        public BigInteger B => fraction.Denominator;
```

Comment:
Was unable to make "Numerator" and "Denominator" display in the order that I wanted (in VS it's alphabetical).

Reply:
I like it.